src/SignalR/clients/java/signalr/signalr.client.java.Tests.javaproj

```diff
@@ -42,15 +42,17 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.targets))\Directory.Build.targets" />
 
   <!-- Define Target overrides after importing Directory.Build.targets so these don't get overridden -->
-  <Target Name="Pack" DependsOnTargets="$(PackDependsOn)" Condition="'$(IsPackable)' == 'true'">
+  <!-- Do not override if SkipTestBuild=true, as we actually want the empty targets that would be set up from the arcade
+       sdk -->
```

Comment:
Please remove this comment because it's misleading. The following never outputs anything because the `Build` target is _always_ overridden. Last definition wins and the `Condition` doesn't matter.
``` msbuild
  <Target Name="Build">
    <Message Text="Hello world" Importance="high" />
  </Target>
  <Target Name="Build" Condition="false" />
```
In any case, why are the empty targets in the Arcade SDK important here?

If they really are "special", extend the comment to say _why_ and this probably needs a `<Choose>` wrapping the `<Target>`.

Reply:
Oops `<Choose>` can't wrap a `<Target>`. Only way to avoid redefining `Pack` target would be to move this into a separate `*.targets` file and make the `<Import />` conditional. But, again, it's not clear why the empty Arcade targets matter.

Reply:
We don't need to redefine it. just keep it from being executed. Will update comment.